circle.yml

```diff
@@ -1,3 +1,11 @@
 machine:
   timezone:
-    Asia/Tokyo
\ No newline at end of file
+    Asia/Tokyo
+test:
+  override:
+    - GOOS=linux GOARCH=amd64 go test -v ./...
+    - GOOS=linux GOARCH=386 go get -v ./...
```

Comment:
No. Because `utmp` struct size is different from host amd64, `TestUsers` will fail.